modules/installing-ocp-agent.adoc

```diff
@@ -51,38 +51,39 @@ cat << EOF > ./my-cluster/install-config.yaml
 apiVersion: v1
 baseDomain: test.example.com
 compute:
-- architecture: amd64
+- architecture: amd64 <1>
   hyperthreading: Enabled
   name: worker
   replicas: 0
 controlPlane:
-  architecture: amd64
+  architecture: amd64 <1> 
```

Comment:
- Reusing callout numbers does not display correctly on the Customer Portal, should avoid this until a fix is available.
- I don't actually understand YAML, but does this line need a `-` in front of `architecture` like line 54?

Reply:
Thats a good point lemme check with that!